styles.css

```diff
@@ -90,29 +101,47 @@ main {
 	max-width: 2000px;
 	height: auto;
 	position: relative;
+	padding: 20px;
 }
 
 header img {
 	width: 100%;
 }
 
 .portfolio {
+	display: flex;
+	flex-flow: column nowrap;
+	border: 1px solid;
+	background-color:var(--dark-blue); 
+	color: var(--my-white);
+	margin: 20px;
 	padding: 20px;
 }
 
+.portfolio-card-container{
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+}
+
 .portfolio-card {
 	border: 1px solid;
 	padding: 20px;
 	margin: 20px 0px;
+	background-color: var(--my-white);
+	color: var(--dark-blue);
+	width: 300px;
 }
 
 .portfolio-card img{
 	height: 200px;
```

Comment:
you might want to constrain the image in another way, eg try adding another picture thats a different aspect ratio and see what happens